tensorboard/webapp/header/reload_container.ts

```diff
@@ -0,0 +1,94 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+import {Component} from '@angular/core';
+import {Store} from '@ngrx/store';
+import {Observable} from 'rxjs';
+import {map} from 'rxjs/operators';
+
+import {State} from '../core/store/core_types';
+import {getPluginsListLoaded} from '../core/store/core_selectors';
+import {manualReload} from '../core/actions';
+import {DataLoadState} from '../types/data';
+
+@Component({
+  selector: 'app-header-reload',
+  template: `
+    <button
+      [class.loading]="isReloading$ | async"
+      mat-icon-button
+      (click)="triggerReload()"
+      [title]="getReloadTitle(lastLoadedTimeInMs$ | async | date: 'medium')"
+    >
+      <mat-icon svgIcon="refresh_24px"></mat-icon>
+    </button>
+  `,
+  styles: [
+    `
+      button,
+      mat-icon {
```

Comment:
Same: selector on "button" and "mat-icon" are potentially confusing but potential is lowered if size of components remains small.